ablySpec/RealtimeClientChannel.swift

```diff
@@ -1297,6 +1299,66 @@ class RealtimeClientChannel: QuickSpec {
 
             }
 
+            context("crypto") {
+                it("if configured for encryption, channels encrypt and decrypt messages' data") {
```

Comment:
I like this test, however I am concerned that at no point are we actually checking that the message is encrypted i.e. if you removed the cipher options completely this test would still pass.  A simple way to make sure this test is actually checking for encrypted messages is to add another subscriber that has encryption disabled and check that the messages are received with an encrypted payload i.e. `data` should be different and `encoding` should indicate it's an encrypted payload